dts/bindings/adc/st,stm32-adc.yaml

```diff
@@ -27,6 +27,14 @@ properties:
     pinctrl-names:
       required: true
 
+    st,prescaler:
+      type: int
+      required: false
+      default: 0
+      description: |
+        Clock prescaler at the input of the adc
+        Could be in range [0 .. 0xF] ; default is 0 (not divided)
```

Comment:
This default needs a justification as per https://docs.zephyrproject.org/latest/guides/dts/bindings.html#default-values-for-properties

Reply:
Ok, default is the reset value